arrabbiata/src/columns.rs

```diff
@@ -59,42 +60,89 @@ impl From<Column> for usize {
     }
 }
 
-pub struct Challenges<F: Field> {
-    /// Challenge used to aggregate the constraints
-    pub alpha: F,
+pub struct Challenges<F> {
+    /// Used to aggregate the constraints describing the relation. It is used to
+    /// enforce all constraints are satisfied at the same time.
+    /// Often noted `α`.
+    pub constraint_randomiser: F,
 
-    /// Both challenges used in the permutation argument
+    /// Both challenges used in the permutation argument.
     pub beta: F,
     pub gamma: F,
 
-    /// Challenge to homogenize the constraints
-    pub homogenous_challenge: F,
+    /// Used to homogenize the constraints and allow the protocol to fold two
+    /// instances of the same relation into a new one.
+    /// Often noted `u` in the paper mentioning "folding protocols".
+    pub constraint_homogeniser: F,
+
+    /// Used by the accumulation protocol (folding) to perform a random linear
+    /// transformation of the witnesses and the public values.
+    /// Often noted `r` in the paper mentioning "folding protocols".
+    pub relation_randomiser: F,
```

Comment:
Note that even though it is a challenge, it must be not accumulated by the accumulation scheme.